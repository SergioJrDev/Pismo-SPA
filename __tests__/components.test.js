import React from 'react';
import renderer from 'react-test-renderer';

import CardFavorito from './../src/shared/favoriteCard'
import Card from './../src/shared/Card'

it('Renderizar componentes compartilhados', () => {
  const componentCardFav = renderer.create(
    <CardFavorito data={{poster: 'img.jpg', show_title: 'Título do filme'}} />
  ).toJSON();
  expect(componentCardFav).toMatchSnapshot();

  const componentCard = renderer.create(
    <Card data={{poster: 'img.jpg', show_title: 'Título do filme'}} />
  ).toJSON();
  expect(componentCard).toMatchSnapshot();
});
