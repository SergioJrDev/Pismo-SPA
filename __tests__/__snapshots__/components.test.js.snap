// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="card"
>
  <figure>
    <img
      alt=""
      src="img.jpg"
    />
  </figure>
  <div
    className="card_content"
  >
    <h2>
      Título do filme
    </h2>
    <a
      className="btn btn-theme btn-icon"
      href="#"
      onClick={[Function]}
    >
      <i
        aria-hidden="true"
        className="fa fa-heart"
      />
    </a>
  </div>
</div>
`;

exports[`renders correctly 2`] = `
<div
  className="card"
>
  <figure>
    <img
      alt=""
      src="img.jpg"
    />
  </figure>
  <div
    className="card_content"
  >
    <h2>
      Título do filme
    </h2>
    <span
      className="descr"
    >
      Elenco
    </span>
    <p />
    <span
      className="descr"
    >
      Diretor
    </span>
    <p />
    <span
      className="descr"
    >
      Sinopse
    </span>
    <p />
    <a
      className="btn btn-theme btn-small"
      href="#"
      onClick={[Function]}
    >
      Adicionar aos favoritos
    </a>
  </div>
</div>
`;
